eng/ci/code-mirror.yml

```diff
@@ -0,0 +1,21 @@
+trigger:
+  branches:
+    include:
+    # Keep this set limited as appropriate (don't mirror individual user branches).
+    - main
+    - release/*
+    - v3.x
+    - v4.x
+
+resources:
+  repositories:
+  - repository: eng
+    type: git
+    name: engineering
+    ref: refs/tags/release
+
+variables:
+  - template: ci/variables/cfs.yml@eng
+
+extends:
+  template: ci/code-mirror.yml@eng
```

Comment:
small nit: end files with an empty newline. (Thats what the red symbol means - empty newline missing)